libs/platform/src/lib/components/menu-button/menu-button.component.html

```diff
@@ -0,0 +1,13 @@
+<button
+    fd-button
+    [options]="menuOptions"
+    [compact]="_iscompact"
```

Comment:
when you prefix something as `_` then it means its private and when something is private you should not really use in the template

Reply:
will remove and use size=='compact' in template.